### PR TITLE
fix: add images to Instagram Feed on Home Page

### LINKED ISSUE #2080: [BUG] Home page instagram feed layout is broken on mobile
**Describe the bug**
Instagram Feed section images on Home page are too big and break the whole page layout.
There should be mobile versions of images implemented.

**How to reproduce**
1. go to Home page on storybook docs 
2. switch to mobile and scroll to the `Share your look` section.  

**Expected behavior**
Images in instagram feed should adjust its size to the mobile mode. There should be adjusted to our designs.

**Actual behavior**

**Screenshots**
hom_page_instagram_feed

**Devices (please complete the following information):**
 - Device: [e.g. iPhone6, Desktop]
 - Browser: [e.g chrome, safari][version]

**Additional context**
Anything you would like to add but was not sure which section it fits. For example: tech stack used (Nuxt, Vue, Gridsome, React, etc)

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue

Closes #2080 

# Scope of work

# Screenshots of visual changes
Add srcsets to SfImage with data for different images.

# Checklist

- [x] No commented blocks of code left
- [ ] Run tests and docs
- [x] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Review by justyna-13 — COMMENTED** [review]
Screenshot from 2021-11-17 09-26-22
On a small mobile screen, images are still a bit too big :)

File: packages/vue/src/components/pages/home/Home.vue [hunk 1]
```diff
@@ -304,32 +320,66 @@ export default {
...
+            desktop: {
+              src: "/assets/storybook/Home/imageAd.webp",
+              width: 470,
+            },
             content: "angelina_trn",
-            url: "/assets/storybook/Home/imageA.png",
           },
           {
+            mobile: {
+              src: "/assets/storybook/Home/imageBm.webp",
```
Comment (aniamusial) [inline packages/vue/src/components/pages/home/Home.vue:335]: praise: great job on using webp :)

File: packages/vue/src/components/pages/home/Home.vue [hunk 2]
```diff
@@ -408,16 +459,32 @@ export default {
         margin: calc(var(--spacer-xl) / 2) 0 0 0;
       }
     }
+    @include for-desktop {
+      &:last-child {
+        .instagram-grid__col:last-child {
+          position: absolute;
+          top: 180px;
```
Comment (aniamusial) [inline packages/vue/src/components/pages/home/Home.vue:466]: suggestion: I would use rems here

**Review by aniamusial — APPROVED** [review]

File: packages/vue/src/components/pages/home/Home.vue (hunk 2 above)
Comment (AdamPawlinski) [reply packages/vue/src/components/pages/home/Home.vue]: I changed it. Thx.

**Comment by AdamPawlinski** [comment]
Width changed to 100% on mobile, so it should look better now.

**Review by justyna-13 — APPROVED** [review]

**Review by aniamusial — APPROVED** [review]

**Pull request merged**